pkg/core/resources/apis/mesh/dataplane_overview.go

```diff
@@ -112,3 +112,14 @@ func (d *DataplaneOverviewResourceList) RetainMatchingTags(tags map[string]strin
 	}
 	d.Items = result
 }
+
+// RetainGatewayDataPlanes to get only gateway Dataplanes
+func (l *DataplaneOverviewResourceList) RetainGatewayDataPlanes() {
```

Comment:
nitpicking: It's spelled `Dataplanes` not `DataPlanes`

at least we do it this way across the project

Reply:
and also can we add code style guide to the repo on which format to use for variables and functions.

Reply:
Variables and Functions casing is forced by Go language (UpperCase for public, lowerCase for private)